scripts/reborn_webui_v2_live_qa/test_run_live_qa.py

```
@@ -252,6 +259,518 @@ def fake_capability_run_statuses(_reborn_home, capability_ids):
+            await action(FakePage())
+
+        async def fake_wait_for_assistant_reply(_page, **_kwargs):
+            return "Slack is connected"
+
+        async def fake_approve_visible_tool_gate(_page):
+            return None
+
+        async def fake_sleep(_seconds):
+            return None
+
+        def fake_capability_run_statuses(_reborn_home, _capability_ids):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `fake_capability_run_statuses`

(ANN202)